### PR TITLE
new kubectl-ui CLI

### PR DESCRIPTION
**Pull request by nerdeveloper**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
 <project version="4">
+  <component name="CommitMessageInspectionProfile">
+    <profile version="1.0">
+      <inspection_tool class="CommitFormat" enabled="true" level="WARNING" enabled_by_default="true" />
+      <inspection_tool class="CommitNamingConvention" enabled="true" level="WARNING" enabled_by_default="true" />
+    </profile>
+  </component>
   <component name="VcsDirectoryMappings">
     <mapping directory="" vcs="Git" />
   </component>
```
Comment (phirmware) [inline .idea/vcs.xml]: git ignore

File: .idea/workspace.xml [hunk 2]
```diff
@@ -1,35 +1,122 @@
...
-  <component name="GOROOT" url="file:///usr/local/go" />
+  <component name="FileTemplateManagerImpl">
+    <option name="RECENT_TEMPLATES">
+      <list>
+        <option value="Go File" />
+        <option value="Go Application" />
+      </list>
+    </option>
+  </component>
+  <component name="GOROOT" url="file:///usr/local/opt/go/libexec" />
```
Comment (phirmware) [inline .idea/workspace.xml]: git ignore also

File: cmd/version.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+	Short: "Print the version number of version",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("version: 1.0.0")
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+
+}
```
Comment (phirmware) [inline cmd/version.go:21]: Is there a need for different files `cmd/version` && `cmd/start` ?

File: config/kubectlui.go [hunk 4]
```diff
@@ -15,7 +15,7 @@ const (
 )
 
 func (k *KubectlUI) setDefaults() {
-	k.v.SetDefault(ApplicationPort, "8080")
+	k.v.SetDefault(ApplicationPort, "3553")
```
Comment (phirmware) [inline config/kubectlui.go:18]: why do we change the default port?

Comment (nerdeveloper) [reply config/kubectlui.go:18]: By default, any web server uses 80, 8080, your app shouldn’t be using such port, nodejs uses 3000 angular uses 4200, Kubernetes uses 6443

File: .idea/workspace.xml (hunk 2 above)
Comment (nerdeveloper) [reply .idea/workspace.xml]: Definitely

File: .idea/vcs.xml (hunk 1 above)
Comment (nerdeveloper) [reply .idea/vcs.xml]: Will do

File: config/kubectlui.go (hunk 4 above)
Comment (phirmware) [reply config/kubectlui.go:18]: got it👌🏽

**Review by phirmware — APPROVED** [review]

**Pull request merged**